facade/v4/facade_structs.go

```diff
@@ -65,62 +65,68 @@ func (f *FacadeV4TaskWaiter[T]) WaitForTaskCompletion() ([]*T, error) {
 	taskStatusValue := v4prismModels.TASKSTATUS_UNKNOWN
 	taskStatus := &taskStatusValue
 
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
thanks for improving this but please keep these changes out of this PR. Create a separate PR for addressing these issues. Keep this PR focused solely on context.